data/paver/skeleton/templates/index.html.template

```diff
@@ -1,9 +1,9 @@
-<!doctype html>
+{% raw %}<!doctype html>
 <html>
-    <head><title>$(( plugin ))$ skeleton page</title></head>
+    <head><title>{{ plugin }} skeleton page</title></head>
```

Comment:
One general note about mixing Angular and Jinja: in general I've come to believe it's a really bad code smell if you're rendering dynamic data server side into an Angular template.  There are times when that's useful but it costs you a lot more in the long run, e.g. it becomes way harder to write meaningful unit tests for such code.  That's part of why we change this; not only do we prefer to not change our default Jinja tags (as Rob says), we don't think it should be necessary in the first place.